lib/temporal/errors.rb

```diff
@@ -47,6 +47,10 @@ def initialize(new_run_id:)
     end
   end
 
+  # Thrown when we can't complete a workflow due to operations occurring after the workflow
+  # is finished.  Could be due to signals, dangling asynchronous activities, or internal framework bug.
+  class TryingToCompleteWorkflowError < InternalError; end
```

Comment:
I think this error name is a bit confusing, what about `WorkflowHasAlreadyStopped`? Or something similar